src/queueUI/Line.java

```diff
@@ -0,0 +1,75 @@
+package queueUI;
+
+import javax.swing.*;
+
+
+public class Line {
```

Comment:
Why is the name of the clas `Line`?

For context:

> There are only two hard things in Computer Science: cache invalidation and naming things.
-- Phil Karlton

(We will talk about cache invalidation in the project)